modules/github/example/variables.tf

```diff
@@ -99,3 +99,7 @@ variable "keyvault_certificate_requests" {
 variable "github_projects" {
   default = {}
 }
+
+variable "jenkins_projects" {
```

Comment:
Why does the github example need a jenkins project?

Reply:
its unnecessary - inherited code form jenkins branch